src/mark.ts

```diff
@@ -99,6 +103,14 @@ export interface MarkConfig extends ColorMixins, VgMarkConfig {
    *
    */
   filled?: boolean;
+
+  /**
+   * The tooltip text string to show upon mouse hover or an object defining which fields should the tooltip be derived from.
+   *
+   * - If `tooltip` is `{"fields": "encodings"}`, then all fields in encodings will be used.
```

Comment:
"all fields from encodings will be used"

Reply:
It's actually all fields from `encoding`.  This makes me if it should be `{"fields": "encodings"}`, suppose we don't change to `{"encoding": true}`